schema.sql

```diff
@@ -75,4 +75,15 @@ CREATE TABLE visits (
     FOREIGN KEY (animal_id) REFERENCES animals (id) ON DELETE RESTRICT ON UPDATE CASCADE,
   CONSTRAINT fk_vets_visits
 	  FOREIGN KEY (vet_id) REFERENCES vets (id) ON DELETE RESTRICT ON UPDATE CASCADE
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+--Update visits table to add id column
+ALTER TABLE visits ADD COLUMN id INT GENERATED ALWAYS AS IDENTITY;
+
+-- Add Non-clustered index for tables
+CREATE INDEX animal_id_index ON visits (animal_id);
+CREATE INDEX vet_id_id_index ON visits (vet_id);
+CREATE INDEX email_id_index ON owners (email);
```

Comment:
- [ ] As per this [preparation requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=Depening%20on%20your,bigger%20than%201000ms.), The execution time in the screenshots BEFORE is significantly lower than 1 sec (1000ms) So Please make The execution time BEFORE >= 1000 ms.
You can do that by running the query multiple times.